src/tests/Regressions/coreclr/GitHub_87879_AppHost/test87879.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Part of the problem may be that the name of the binary used by the apphost test is same name of the binary used by the non-apphost test. You may want to rename the .csproj to test87879__apphost.csproj to give it a different name.

Reply:
Hmm, that didn't seem to do it either... are we sure the normal test is even running?

Reply:
Yes, the normal test is running. For example, here is the test log from `Build coreclr Pri0 Runtime Tests Run windows x64 checked`.

https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-98007-merge-df32ef18c2e64b8ab9/Regressions/1/coreclr/GitHub_87879/test87879/output.txt?helixlogtype=result

```
BEGIN EXECUTION
 "C:\h\w\BB7B09CB\p\corerun.exe" -p "System.Reflection.Metadata.MetadataUpdater.IsSupported=false" -p "System.Runtime.Serialization.EnableUnsafeBinaryFormatterSerialization=true"  test87879.dll 
Main thread succeeded.
Secondary thread succeeded.
Expected: 100
Actual: 100
END EXECUTION - PASSED
PASSED
Test Harness Exitcode is : 0
```

Reply:
Do you see the apphost test running if you do a local test run?

Reply:
> If you are okay with it, I can just remove it.

Yes, could you please remove it?

> We could potentially just read the stack size from the AppHost directly on Windows as a part of the main test,

The core runtime tests are not a good place to test shipping apphost.